### PR TITLE
Broadside (ECS deploys) and Aleph (Redshift query GUI)

### PR DESCRIPTION
**Pull request by apurvis**
## Describe Why This Is Awesome

[Broadside](https://github.com/lumoslabs/broadside) is a tool that dramatically simplifies the deployment and configuration of ECS based containerized application.  We have been using it internally for a while and recently open sourced the tool.  [Read more in the release blog post](http://engineering.lumosity.com/amazon%20web%20services/amazon%20elastic%20compute%20cloud/amazon%20ec2%20container%20service/deploys/docker/containers/2017/01/23/broadside-is-a-tool-for-managing-ec2-containers-in-amazon-web-services/).

[Aleph](https://github.com/lumoslabs/aleph) is a full featured web application for writing, running, saving, notarizing, and doing other stuff with your redshift queries.  [Read more in the release post](http://engineering.lumosity.com/data/analytics/business%20intelligence/redshift/2016/06/17/introducing-aleph/)

--

Like this pull request?  Vote for it by adding a :+1:

### CODE REVIEW

**Comment by camertron** [comment]
:+1:

**Comment by apurvis** [comment]
@donnemartin done

File: README.md [hunk 1]
```diff
@@ -796,6 +796,8 @@ AWS Repos:
 
 Community Repos:
 
+* [Lumoslabs/aleph](https://github.com/lumoslabs/aleph) - Collaborative tool for query devepoment. Full featured web application for writing, running, explorign, and revision tracking across an entire organziation datbases.  Currently is focused on Redshift but has generic JDBC suppor and could work anwyere.  Only a month months old in open source but Has traction outside at other notable companies.
```
Comment (donnemartin) [inline README.md]: I'm not sure about the new description change that went in?  I feel it is quite long and has multiple typos.

The requested changes were to fix the name and sorting, could we focus on those?  Thanks!

**Comment by apurvis** [comment]
should be simplified now

**Comment by donnemartin** [comment]
@apurvis thank you!

**Pull request merged**